unit-tests/tests/03_txn_setup_account.test_ex.js

```diff
@@ -0,0 +1,45 @@
+import path from "path";
+import { 
+  emulator, 
+  init, 
+  shallPass,
+  sendTransaction,
+  deployContractByName,
+  getServiceAddress,
+} from "flow-js-testing";
+
+jest.setTimeout(10000);
+
+describe("transactions test", () => {
+  // Instantiate emulator and path to Contracts
+  beforeEach(async () => {
+    const basePath = path.resolve(__dirname, "../../");
+    const port = 8080;
+    await init(basePath, { port });
+    return emulator.start(port);
+  });
+
+  // Stop emulator
+  afterEach(async () => {
+    return emulator.stop();
+  });
+
+  test("setup account", async () => {
+    //Deploying contracts used in setup account
+    await deployContractByName("Profile");
+
+    const serviceAccount = await getServiceAddress();
+    await deployContractByName("NonFungibleToken");
+    const nftAddrMap = { NonFungibleToken: serviceAccount }
+    
+    await deployContractByName("FungibleToken");
+    const ftAddrMap = { FungibleToken: serviceAccount }
+
+    await deployContractByName({name: "NFTStorefront", addressMap: nftAddrMap, ftAddrMap});
+    await deployContractByName({name: "DapperUtilityCoin", addressMap: ftAddrMap});
+
+    await shallPass(async () => {
+      return sendTransaction({ name: "setup_account" });  

```

Comment:
@JawwadNFTGenius When you execute a transaction the blockchain state is changed in some way. Your test **should** validate that the intended change has actually been applied. 

The setup_account transaction creates the public collections under the user account, you must first ensure that the collections did not exist before the transaction was executed and then verify that they were created correctly.